sls-public-demo.html

```diff
@@ -1,44 +1,297 @@
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li>
+
+<p class="first">To import data directly into the PUBLIC group, in the Import Location window choose <span class="bold">PUBLIC</span> from the <span class="bold">Group</span> drop-down menu.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/slsPublicImport.jpg" class="align-center" src="images/slsPublicImport.jpg" style="width: 70%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<li>
+
+<p class="first">Images IDs, shown in the right pane, are used to link to images in the PUBLIC group.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/slsPublicID.jpg" class="align-center" src="images/slsPublicID.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3><a class="anchor" id="syntax"></a>Syntax for Links</h3>
+
+<li>
+
+<p class="first">To create an HTML link to the thumbnail or the full viewer display of the image, use the following syntax where <span class="red">1234</span> represents the image ID:</p>
 
 <p>HTML to embed the thumbnail:</p>
-<p class="code-example">src="https://nightshade.openmicroscopy.org/webgateway/render_thumbnail/3925540/"</p>
+<p class="code-example">src="https://nightshade.openmicroscopy.org/webgateway/render_thumbnail/<span class="red">1234<span>/"</p>
+
+<p>Thumbnails have a default width of 64 px but can be displayed at any width by customising the thumbnail URL with the required size in pixels at the end of the URL:</p>
+
+<p class="code-example">src="https://nightshade.openmicroscopy.org/webgateway/render_thumbnail/<span class="red">1234<span>/200/"</p>
+
+<p>HTML for the link to the full image in the image viewer:</p>
+
+<p class="code-example">src="https://nightshade.openmicroscopy.org/webgateway/img_detail/<span class="red">1234<span>"</p>
+
+<p>If an image in the PUBLIC group has a file attachment, this can be downloaded using a link with the syntax:</p>
+    
+<p class="code-example">href="https://nightshade.openmicroscopy.org/webclient/annotation/358508/"</p>
```

Comment:
Ah, I see the annotations have their own IDs as shown lower down the page, maybe remove the example here and just leave the one which shows where the ID has come from under point. 9?